solutions/observability/apm/collect-metrics.md

```diff
@@ -40,7 +40,7 @@ See the [Open Telemetry Metrics API](https://github.com/open-telemetry/opentelem
 Use **Discover** to validate that metrics are successfully reported to {{kib}}.
 
 1. Open your Observability instance.
-2. Find **Discover** in the main menu or use the [global search field](/get-started/the-stack.md#kibana-navigation-search).
```

Comment:
I think we need to find where this content went rather than just removing the link.

Reply:
Found and updated!